part-1-ajax/test.js

```diff
@@ -1,76 +1,91 @@
-function getCountryName(code) {
-    $.ajax({
-      //uses the three letter code to get the country's full name
-      }
-    })
-  }
 
-  function parseData(country) {
-    //takes the country data
-    //extracts the name, capital, population, and neighboring countries
-    //iterates through the neighbors array to send off second AJAX request
-  }
 
   var button = document.getElementsByTagName('button')[0]
-  button.addEventListener('click', function(){
 
+  button.addEventListener('click', function(event){
+    event.preventDefault()
```

Comment:
you forgot to clear the information on the page. when you search for another country, it just adds to the information already on the page.